PaintTest/main.c

```diff
@@ -0,0 +1,705 @@
+#pragma region Copyright (c) 2014-2016 OpenRCT2 Developers
```

Comment:
Don't call two files `main.c` and `main.cpp` this will cause VS to get confused and delete the object file of whichever one was compiled first.